python/cudf/cudf/core/udf/row_function.py

```diff
@@ -24,6 +26,8 @@
     _supported_dtypes_from_frame,
 )
 
+dstring = DString()
```

Comment:
Does this constructor cause a device allocation or any noteworthy side effects (device/stream synchronization)? If so -- this object is only used to get the size, from what I can tell, so we might want to think about another way to get that information.

Reply:
Nope, thankfully no device side allocation here.